test/e2e/generator/generate.go

```diff
@@ -323,6 +326,14 @@ func generateNode(
 
 	if startAt > 0 {
 		node.StateSync = nodeStateSyncs.Choose(r)
+		if manifest.InitialHeight-startAt <= 5 && node.StateSync == e2e.StateSyncDisabled {
+			// avoid needint to blocsync more than five
+			// total blocks.
```

Comment:
```suggestion
			// avoid needing to blocsync more than five total blocks.
```

Is "five" just "a small number so it doesn't take too long?

Reply:
Each time we start a node in the future, it's +5 of the initial height or the last point that we started a node, so this is basically saying "don't do blocksync for more than one standard interval